content/patterns/fields.zh.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "字段模板"
```

Comment:
我考虑过，但是我看文中基本给的都是一些例子，所以就改了一下 改成模板，会比较容易理解我感觉。

我考虑下

Reply:
I wouldn't necessarily call them best practice, some of these are actually work arounds until future enhancements to CUE are available. Some are neat tricks, more like "useful or interesting" patterns. They came from "repeated" solutions to certain questions that have come up again and again in the CUE community.

Reply:
Could we just call it as template in Chinese translation? Or snippet? The code can be repeated in more complicated use cases. Right?

Because the 'pattern' is harder to understand in Chinese.